casper/src/main/scala/coop/rchain/casper/util/comm/DeployService.scala

```diff
@@ -16,7 +16,7 @@ trait DeployService[F[_]] {
   def createBlock(): F[Option[BlockMessage]]
   def showBlock(q: BlockQuery): F[String]
   def showBlocks(): F[String]
-  def addBlock(b: BlockMessage): F[Unit] //add a block to Casper internal state
+  def addBlock(b: BlockMessage): F[(Boolean, String)] //add a block to Casper internal state
```

Comment:
Why not `Either` so that it's consistent with casper?

Reply:
to keep aligned with `def deploy(d: DeployString): F[(Boolean, String)]` in the same trait.
@birchmd WDYT which should be preferred?

Reply:
Reusing the deploy return type is good with me. Idea was that `_1` is a simple success/failure Boolean and `_2` is a message giving more information. And this is exactly the same as gRPC `DeployServiceResponse`